main_test.go

```diff
@@ -26,6 +26,7 @@ func Test_realmain(t *testing.T) {
 		wantLog string
 		want    int
 	}{
+		{"multi EH", []string{"couper", "run", "-f", base + "/16_couper.hcl"}, nil, `duplicate error type registration: \"*\";`, 1},
```

Comment:
Please move this to `error_handler` related test files. `main_test` should apply to Couper binary related options.
I have seen additional tests which sneaked into this test file. All of them are unrelated to `main.go`.

Reply:
done